smarturls.py

```diff
@@ -19,6 +19,7 @@
     "int4": "\d{4,4}",
     "slug": "[\w-]+",
     "username": "[\w.@+-]+",
+    "uuid": "[0-9a-f]{32}",
```

Comment:
We should also support UUIDs like: de305d54-75b4-431b-adb2-eb6b9e546014, according to wikipedia this is canonical format.